service/src/main/java/org/apache/griffin/core/job/SparkSubmitJob.java

```diff
@@ -120,10 +133,11 @@ private String post2Livy() {
             HttpHeaders headers = new HttpHeaders();
             headers.setContentType(MediaType.APPLICATION_JSON);
             headers.set(REQUEST_BY_HEADER,"admin");
-          
-            HttpEntity<String> springEntity = new HttpEntity<String>(toJsonWithFormat(livyConfMap), headers );
+
+            HttpEntity<String> springEntity = new HttpEntity<String>(toJsonWithFormat(livyConfMap),
+                    headers);
             result = restTemplate.postForObject(livyUri,springEntity,String.class);
```

Comment:
why remove headers?

Reply:
When griffin-check is used, this line becomes multi-line.

Reply:
Yes,this idea is very important, now use SingleThreadExecutor to manage single-threaded tasks, preventing single-threaded hangs.